flow-libs/image-size.js.flow

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+declare module 'image-size' {
+  declare module.exports: (input: Buffer) => { width: number, height: number, ...};
```

Comment:
Apparently, `width` and `height` can also be undefined: https://github.com/image-size/image-size/blob/a38b56fe7303898ce6811adb213e518a57593d10/lib/types/interface.ts#L1-L10

Reply:
I don't think those types match the implementation (I believe there should be union types of the different combinations), but better to match it.